### DISCUSSION TITLE
Need for a paid support plan

### DISCUSSION THREAD

**Discussion by shibayan** [polymind-inc/acmebot#537] [Polls]
**IMPORTANT**: Acmebot will never be a paid service. Questions are for the support portion only.

Some users had requirements that were beyond Acmebot's support as OSS. I would like to help, but it is difficult because it would be endless and take away from my resources.

However, I am wondering if a paid support plan would improve this situation.

**Comment by shibayan** [comment]
I'm preparing to offer a paid support plan. Please wait for a while.

**Reply by yarashagarwal** [reply]
Hi, Do you have any further information on the paid support plan?

**Reply by shibayan** [reply]
We are not ready yet. If you are in a hurry, sponsorship may provide priority support.
https://github.com/sponsors/shibayan
